frontend/packages/kubevirt-plugin/src/components/modals/disk-modal/disk-modal.tsx

```diff
@@ -370,19 +379,34 @@ export const DiskModal = withHandlePromise((props: DiskModalProps) => {
               />
             </FormRow>
           )}
-          <FormRow title="Interface" fieldId={asId('interface')} isRequired>
+          <FormRow
+            title="Interface"
+            fieldId={asId('interface')}
+            isRequired
+            validation={busValidation}
+          >
             <FormSelect
               onChange={React.useCallback((diskBus) => setBus(DiskBus.fromString(diskBus)), [
                 setBus,
               ])}
+              isValid={!isValidationError(busValidation)}
               value={asFormSelectValue(bus)}
               id={asId('interface')}
               isDisabled={inProgress}
             >
               <FormSelectPlaceholderOption isDisabled placeholder="--- Select Interface ---" />
-              {DiskBus.getAll().map((b) => (
-                <FormSelectOption key={b.getValue()} value={b.getValue()} label={b.toString()} />
-              ))}
+              {!validAllowedBusses.has(bus) && (
+                <FormSelectOption
+                  key={bus.getValue()}
+                  value={bus.getValue()}
+                  label={`${bus.toString()} --- Invalid ---`}
```

Comment:
Ok, I suppose the user should know it is not selectable 

We could try to make the disabled work in the future